xinference/model/llm/llm_family_modelscope.json

```diff
@@ -369,6 +369,105 @@
       ]
     }
   },
+  
+  {
+    "version": 1,
+    "context_length": 2048,
+    "model_name": "xverse-chat",
+    "model_lang": [
+      "en",
+      "zh"
+    ],
+    "model_ability": [
+      "chat"
+    ],
+    "model_description": "XVERSE-7B is a multilingual large language model, independently developed by Shenzhen Yuanxiang Technology.",
+    "model_specs": [
+      {
+        "model_format": "pytorch",
+        "model_size_in_billions": 7,
+        "quantizations": [
+          "4-bit",
+          "8-bit",
+          "none"
+        ],
+        "model_hub": "modelscope",
+        "model_id": "xverse/XVERSE-7B-Chat",
+        "model_revision": "v1.0.0"
+      },
+      {
+        "model_format": "pytorch",
+        "model_size_in_billions": 13,
+        "quantizations": [
+          "4-bit",
+          "8-bit",
+          "none"
+        ],
+        "model_hub": "modelscope",
+        "model_id": "xverse/XVERSE-13B-Chat",
+        "model_revision": "v1.0.0"
+      }
+    ],
+    "prompt_style": {
+      "style_name": "XVERSE",
+      "system_prompt": "",
+      "roles": [
+        "Human",
+        "Assistant"
+      ]
+    }
+  },
+  {
+    "version": 1,
+    "context_length": 2048,
+    "model_name": "xverse",
+    "model_lang": [
+      "en",
+      "zh"
+    ],
+    "model_ability": [
+      "generate"
+    ],
+    "model_description": "XVERSE is a multilingual large language model, independently developed by Shenzhen Yuanxiang Technology.",
+    "model_specs": [
+      {
+        "model_format": "pytorch",
+        "model_size_in_billions": 7,
+        "quantizations": [
+          "4-bit",
+          "8-bit",
+          "none"
+        ],
+        "model_id": "xverse/XVERSE-7B",
+        "model_hub": "modelscope",
+        "model_revision": "v1.0.0"
```

Comment:
I  think you'd better to use `master`, since `master` branch was updated 14 days ago, while tag `v1.0.0` was updated 1 month ago.
Check all the other places.